File: docs/legacy(old PACE)/unsupported/VB.NET/PaceWebServiceSDK/sample/ListPaceConnectTypes.vb
Imports PaceWebServiceSDK.efipaceservices

Public Class ListPaceConnectTypes
    Public Shared Sub Run()
        Dim invokePaceConnectHttpBinding As InvokePaceConnectHttpBinding = PaceClient.getInvokePaceConnectHttpBinding()
        Dim paceConnectTypes As String() = invokePaceConnectHttpBinding.getPaceConnectTypes()

        For Each paceConnectType As String In paceConnectTypes
            Console.WriteLine("Type : " + paceConnectType)
        Next

    End Sub
End Class

File: docs/legacy(old PACE)/C#/4.0/PaceAPISample/sample/ListPaceConnectTypes.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using efipaceservices;

namespace Pace_Web_Service_SDK.sample
{
    class ListPaceConnectTypes
    {
        public static void Run()
        {
            InvokePaceConnectHttpBinding invokePaceConnectHttpBinding = PaceClient.getInvokePaceConnectHttpBinding();
            String[] paceConnectTypes = invokePaceConnectHttpBinding.getPaceConnectTypes();
            foreach (String paceConnectType in paceConnectTypes)
            {
                Console.WriteLine("Type : " + paceConnectType);
            }
        }
    }
}
